vidarr-server/src/test/java/ca/on/oicr/gsi/vidarr/server/PriorityByWorkflowTest.java

```diff
@@ -0,0 +1,147 @@
+package ca.on.oicr.gsi.vidarr.server;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import ca.on.oicr.gsi.vidarr.ConsumableResourceResponse;
+import ca.on.oicr.gsi.vidarr.ConsumableResourceResponse.Visitor;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.Optional;
+import org.junit.Before;
+import org.junit.Test;
+
+public class PriorityByWorkflowTest {
+
+  private PriorityByWorkflow sut;
+  private ObjectMapper mapper = new ObjectMapper();
+
+  String workflow = "test";
+  String version = "1.0";
+  ConsumableResourceResponse.Visitor consumableResourceCheckerVisitor = new Visitor<Optional<String>>() {
```

Comment:
raw use of parameterized class - should be ConsumableResourceResponse.Visitor\<Optional\<String\>\>, and then you can use `= new Visitor<>` and the casts throughout this file become redundant